### DISCUSSION TITLE
Disable Props from Being Displayed in "data-page" attribute

### DISCUSSION THREAD

**Discussion by digitalam025** [inertiajs/inertia#695] [Help] [answered]
Hi! I'm working with Inertia (Vue3) and Laravel as the backend. Every prop passed through from the backend is being displayed in a `<div> ` attribute like so 

![image](https://user-images.githubusercontent.com/46593240/119802160-e6f30400-bf10-11eb-8873-dd9c2edfc65e.png)

Is there a way to disable this? Thanks in advance!

**Comment by ajnsn** [comment, marked as answer]
Hi @digitalam025, 

there's already a thread about this topic [over here](https://github.com/inertiajs/inertia/discussions/372). 

If you want oo disable the inheritance, e.g. the errors prop, you can set `inheritAttrs` to `false`. Please also see the PR [over here](https://github.com/inertiajs/inertia/pull/681). It changes the overall behaviour of the Vue 3 adapter but is not released yet.

**Reply by digitalam025** [reply]
Ah brilliant! Thanks! :)

**Reply by abdorrahmani** [reply]
Hi, I have similar problem, how to disable the inheritance, e.g. the errors prop in React? Thanks

**Comment by irfanahmad7272** [comment]
I am facing similar issue, using vue2, and this data is visible as Json objects when I test send a GET request to my site. I see as Json objects there, that I passed as props. How can i hide so that no one can crawl my site pages and view this as json objects? Thanks
